e2e/applitools-utils.js

```diff
@@ -0,0 +1,19 @@
+import { screenSizeDesktop, screenSizeMobile } from './cypress-utils'
+
+const eyesCheck = (name, mobile) => {
+  const { width, height } = mobile ? screenSizeMobile : screenSizeDesktop
+
+  const appName = 'Embed Widget - New'
+  cy.eyesOpen({
+    appName,
+    batchName: appName,
+    batchId: 'matej123',
```

Comment:
if we leave this we don't need the ENV variable right? (since is a random ID I think it's better leave your id across the app)

Reply:
Correct. `batchId` does not seem to affect anything. I want to try running this in Travis to see what happens. This PR is still a work in progress.

Reply:
ah ok I approved it anyway, didn't see anything wrong